auditlog/models/rule.py

```diff
@@ -364,11 +472,20 @@ def create_logs(self, uid, res_model, res_ids, method,
         http_session_model = self.env['auditlog.http.session']
         for res_id in res_ids:
             model_model = self.env[res_model]
+            # Do extra check for active_model situations where res_model
+            # is not preloaded in auditlog model_cache
+            model_id = self.pool._auditlog_model_cache.get(res_model, False)
```

Comment:
False as a default is redundant. If the model is not present, dict.get() returns None. Maybe make this a little more compact as
```
self.pool._auditlog_model_cache.get(
    res_model,
    self.env['ir.model'].search([('model', '=', res_model)]).id)
```
you will lose the check on whether the model exists but a missing model was going to error out in line 474 anyway.